src/components/FilteringPanel/TodosLeft/TodosLeft.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { Todo } from '../../../types/Todo';
+
+type TodosLeftProps = {
+  todos: Todo[];
+};
+
+export const TodosLeft: React.FC<TodosLeftProps> = ({ todos }) => (
+  <span className="todo-count" data-cy="TodosCounter">
+    {todos.filter(todo => !todo.completed).length} items left
+  </span>
+);
```

Comment:
it's redundant to create separate component like this one

we usually create new component:
1. if we want to reuse them somewhere several times;
2. or if this component is too big, to simplify the logic